app/screens/login/login.js

```diff
@@ -143,6 +143,14 @@ export default class Login extends PureComponent {
         Keyboard.dismiss();
         InteractionManager.runAfterInteractions(async () => {
             if (!this.props.loginId) {
+                t('login.noEmail');
```

Comment:
Can this be done in a separate PR? It's not related to the other changes

Reply:
yes I can move it to a separate PR